src/__tests__/__snapshots__/Box.test.tsx.snap

```diff
@@ -49,6 +49,18 @@ exports[`Box renders margin 3`] = `
   margin: 4px;
 }
 
+@media screen and (min-width:544px) {
+
+}
+
+@media screen and (min-width:768px) {
+
+}
+
+@media screen and (min-width:1012px) {
+
+}
```

Comment:
Not sure what's causing these to get output

Reply:
Seems like this is a know issue both [for us](https://github.com/primer/react/issues/1298) and for [the community](https://github.com/styled-components/jest-styled-components/issues/276) 

That said since we are moving away from jest snapshots ([ADR reference](https://github.com/primer/react/blob/main/contributor-docs/adrs/adr-011-snapshot-tests.md#decision)), if this is only snapshot issue (seems like it) we might maybe want to start deleting these snapshots? At least we can start with Box maybe? And if needed we can create a test in Playwright. What do you think?